lib/housekeeping.js

```diff
@@ -37,18 +38,148 @@ class HouseKeeper {
     this.monitor = monitor;
     this.tagger = tagger;
   }
+  
+  _calculateVolumeTotals(tabulatedVolumes) {
+    let totalVolumes = {};
+    
+    Object.keys(tabulatedVolumes).forEach(function(volumesInRegion) {
+      tabulatedVolumes[volumesInRegion].forEach(function(volume) {        
+        let region = volume.AvailabilityZone;
+        let type = volume.VolumeType;
+        
+        if (!(region in totalVolumes)) {
+          totalVolumes[region] = {};
+        }
+        
+        if (!(type in totalVolumes[region])) {
```

Comment:
Given that all this initialization is required, it might be worth pre-initializing `totalVolumes` before iterating through all the volumes per region, so you can just increment the counts while iterating.

Reply:
Good call. That would definitely clean things up.

Reply:
Any suggestions on how to initialize the volume types? Currently, the possible EBS volume types are `gp2`, `io1`, `st1`, `sc1`, and `standard`. It's probably best not to assume that these EBS volume types will always be the only volume types in the future (perhaps, AWS will add more). The only way I can think of getting all these volume types is to traverse through all of our volumes, but at that point, we might as well not initialize them.

Reply:
Oh crud, I forgot about that. That's not ideal. You're right, we may as well just not initialize them. I guess there's still utility in pre-initializing the region objects, though, right?